bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Widget.java

```diff
@@ -1039,7 +1039,7 @@ public void removeListener (int eventType, Listener listener) {
  * @noreference This method is not intended to be referenced by clients.
  * @nooverride This method is not intended to be re-implemented or extended by clients.
  */
-protected void removeListener (int eventType, SWTEventListener listener) {
+protected void removeListener (int eventType, EventListener listener) {
```

Comment:
As it is a protected method of a public class I think this must stay and be deprecated.

Then we need a new `protected void removeTypedListener (int eventType, EventListener listener) {`

Reply:
Done, the method getting SWTEventListener is marked as deprecated forRemoval.